.github/workflows/ci-pipeline.yml

```diff
@@ -0,0 +1,18 @@
+name: Continuous Integration 
+on: 
+  push:
+    paths-ignore:
+      - 'README.md'
+      - 'documentation/**'      
+  pull_request:
+    paths-ignore:
+      - 'README.md'
+      - 'documentation/**'
+jobs:
+  build:
+    name: Build and test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - run: docker build --target test --tag todo-app:test .
+    - run: docker run --env-file ./.env.test todo-app:test
```

Comment:
It would be nice to document these commands in your Readme so future devs know they can run the tests within Docker too, since they won't necessarily know to look here